overlord/snapstate/snapstate_test.go

```diff
@@ -7118,7 +7118,9 @@ func (s *snapmgrTestSuite) TestUpdateManyAutoAliasesScenarios(c *C) {
 
 		updates, tts, err := snapstate.UpdateMany(context.Background(), s.state, scenario.names, s.user.ID, nil)
 		c.Check(err, IsNil)
-		verifyLastTasksetIsReRefresh(c, tts)
+		if scenario.update {
+			verifyLastTasksetIsReRefresh(c, tts)
```

Comment:
While on it, can we extend verifyLastTasksetIsReRefresh() to check that it has no halt tasks?

Reply:
done